.github/workflows/cypressoldapp.yml

```
@@ -0,0 +1,114 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: app/client/package.json
+      - name: Install dependencies
+        working-directory: app/client
+        run: |
+          yarn install --immutable
+      - name: Setting up the cypress tests
+        if: steps.run_result.outputs.run_result != 'success'
```

actionlint (1.7.4)

property "run_result" is not defined in object type {}

(expression)